sale_report_partner_category/__manifest__.py

```diff
@@ -0,0 +1,17 @@
+# © 2016 Houssine BAKKALI, Coop IT Easy SCRL
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    'name': "Sale Report Partner Category",
+    'summary': """
+        This module extends the sale report to extra fields.
+    """,
+    'author': 'Houssine BAKKALI, Coop IT Easy SCRL',
```

Comment:
Could you: 
- change the author to: `Coop IT Easy SCRLfs` 
- place `Houssine BAKKALI <houssine@coopiteasy.be>` in  `readme/CONTRIBUTORS.rst`
- add the `"website": "https://coopiteasy.be",`